File: ProjectTest/OneLang/PHP/Utils/ArrayHelper.php
<?php

namespace Utils\ArrayHelper;

class ArrayHelper {
    static function sortBy<T>($items, $keySelector) {
        // @java-import java.util.Arrays
        // @java Arrays.sort(items, (a, b) -> keySelector.apply(a) - keySelector.apply(b));
        // @java return items;
        return $items->sort(function ($a, $b) { return $keySelector($a) - $keySelector($b); });
    }
    
    static function removeLastN<T>($items, $count) {
        // @java items.subList(items.size() - count, items.size()).clear();
        $items->splice(count($items) - $count, $count);
    }
}

File: ProjectTest/OneLang/CSharp/Utils/ArrayHelper.cs
using System;
using System.Collections.Generic;

namespace Utils
{
    public class ArrayHelper {
        public static T[] sortBy<T>(T[] items, Func<T, int> keySelector)
        {
            return items.sort((a, b) => keySelector(a) - keySelector(b));
        }
        
        public static void removeLastN<T>(List<T> items, int count)
        {
            items.splice(items.length() - count, count);
        }
    }
}